### ISSUE TITLE
Annotated Case preventing Word from being able to open it

### ISSUE THREAD

**Issue by bjohnson747** [harvard-lil/h2o#640]
I got an error when trying to open an exported Casebook, and narrowed down the item with the issue to this one: https://opencasebook.org/casebooks/75344-contracts-casebook-digital-version/resources/2.2.3.8-the-trustees-of-boston-college-v-the-big-east-conference#

Exported fine, but gets Word error when attempting to open (attached)
<img width="413" alt="screen shot 2019-01-23 at 4 09 45 pm" src="https://user-images.githubusercontent.com/12850389/51637278-53b4f880-1f29-11e9-9f43-d4a5f3c7b553.png">

**Comment by bjohnson747** [comment]
Another casebook that's throwing the same error when I try to export in full: https://h2o-dev-test.lil.tools/casebooks/101663-food-law-casebook/layout#
I haven't dug into it to find out which resource is the problem one.

**Comment by cgruppioni** [comment]
This will stop happening when the next iteration of export code is done, but last resort: let it fail gracefully and email admins

**Comment by cgruppioni** [comment]
Adding notes from ticket #621 

Two examples of failure:

https://opencasebook.org/casebooks/110237-open-property-brett-clone-for-bug-testing/resources/1.8.4-the-defeasible-fees/annotate

https://opencasebook.org/casebooks/110237-open-property-brett-clone-for-bug-testing/resources/1.8.5-remainders-part-4-divestment-of-future-interests-types-of-vested-remainders/annotate

Both can be exported, but give the below errors when attempting to open in word:
screen shot 2018-12-11 at 6 57 06 pm
screen shot 2018-12-11 at 6 56 59 pm

Both contain text that was pasted into H2O.

In separate testing - not on these items, which i've preserved - when I
(1) removed both the footnote links and the line-break (not sure what it is called, it shows up as a red-dotted line in the editor when i put my mouse over it) that's next to the footnotes, and
(2) pasted it into Text Edit, then pasted it back into H2O,
it export successfully.

**Comment by bjohnson747** [comment]
I myself had case that was preventing Word from opening export -

When I removed the center tags that were in my case in question, it exported correctly.

It appears that BR tags are being ignored that are in the HTML, though them being ignored aren't preventing Word from being able to open the export. P tags may also be being ignored within blockquotes?

@cgruppioni FYI!

**Comment by rebeccacremona** [comment]
This error message means that the docx xml is invalid. There are now only two conditions under which we might produce invalid docx xml: a) we have an error in our Table of Contents-producing LUA filter, which inserts XML straight into the output, or b) we've uncovered an error in the pandoc docx writer. Both should be vanishingly rare. Closing this ticket: should we uncover new instances of invalid DOCX, we can open new tickets.
